documentation/wiki/ChangeWaves-Dev.md

```diff
@@ -1,12 +1,10 @@
 ⚠ This doc is intended for internal teams.
```

Comment:
Other comments for this doc:
Line 29/27: missing semicolon
"Creating a change wave" shouldn't talk about end-of-life procedures (3 and part of 4).
For the End-of-Life section:
Title should have hyphens, "step 1" should clarify step 1 of creating a change wave, and step 2 should just say `$([MSBuild]::AreFeaturesEnabled` to match the format of the previous part (or they should both include a version, but I think that's the worse answer).

Reply:
> and step 2 should just say $([MSBuild]::AreFeaturesEnabled to match the format of the previous part (or they should both include a version, but I think that's the worse answer).
agreed, the version is irrelevant there.